azure-iot-device/azure/iot/device/iothub/pipeline/pipeline_stages_iothub_mqtt.py

```diff
@@ -103,8 +106,10 @@ def _run_op(self, op):
                     request_id=op.request_id,
                 )
                 payload = json.dumps(op.request_body)
-                self.continue_with_different_op(
-                    original_op=op, new_op=pipeline_ops_mqtt.Publish(topic=topic, payload=payload)
+                operation_flow.delegate_to_different_op(
+                    self,
```

Comment:
here and everywhere - more than 3 args -> use kwargs. Yes, even for self. `stage=self`